src/main/java/com/tstepnik/planner/domain/User.java

```diff
@@ -10,6 +11,7 @@
 
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "USER_ID")
```

Comment:
Remove this, we dont need it. Also don't change column names like this (id -> user_id). Column names should be lowercase as well.